.github/workflows/ci.yaml

```
@@ -27,17 +41,20 @@ jobs:
       run:
         shell: bash
     steps:
-      - uses: actions/checkout@v4
+      - name: Download pre-built packages
+        uses: actions/download-artifact@v4
+        with:
+          name: Packages
+          path: dist
+      - run: tar xf dist/*.tar.gz --strip-components=1
       - uses: actions/setup-go@v5
         with:
           go-version-file: go.mod
       - uses: actions/setup-python@v5
         with:
           python-version: 3.12
       - name: Install Python dependencies
-        run: |
-          python -m pip install --no-cache uv
-          python -m uv pip install '.'
+        run: python -m pip install --no-cache uv setuptools_scm
```

```suggestion
        run: python -m pip install --no-cache uv && python -m uv pip install setuptools_scm
```

maybe like this? not a huge difference though

---

I've changed this so we no longer need to install any Python deps unless we want to run Python tests.